src/OpenSpaceToolkit/Astrodynamics/Trajectory/Model/Static.cpp

```diff
@@ -18,6 +18,10 @@ Static::Static(const Position& aPosition)
     : Model(),
       position_(aPosition)
 {
+    if (aPosition.accessFrame() != Frame::ITRF())
+    {
+        throw ostk::core::error::runtime::Wrong("Position Frame", aPosition.accessFrame()->getName());
+    }
```

Comment:
While you're at it, can you add a docstring to this class to indicate that this for static positions on the earth's surface (hence why the frame needs to be in ITRF), cause I never really knew what this class was for

https://github.com/open-space-collective/open-space-toolkit-astrodynamics/blob/main/include/OpenSpaceToolkit/Astrodynamics/Trajectory/Model/Static.hpp